app/components/channel_list/categories/categories.tsx

```diff
@@ -20,23 +21,27 @@ const styles = StyleSheet.create({
     },
 });
 
-const renderCategory = (data: {item: CategoryModel}) => {
-    return (
-        <>
-            <CategoryHeader category={data.item}/>
-            <CategoryBody category={data.item}/>
-        </>
-    );
-};
-
 const Categories = (props: Props) => {
```

Comment:
It's not a standard/convention but all our components destructure its props at this line and thus we avoid `props.categories`.  0/5 on this one.